src/index.ts

```diff
@@ -41,17 +50,40 @@ async function start() {
         registerPublicHandlers(SERVICE_NAME, inventoryPublicHandlers);
 
         process.on("SIGINT", () => {
-            console.log("[ITEM-STORE] Gracefully shutting down...");
+            logModel.date = new Date();
+            logger.log({
+                level: 'info',
+                logInfos: logModel,
+                message: "Gracefully shutting down...",
+                correlationId: '123',
+            });
             shutdown(0);
         });
 
         process.on("SIGTERM", () => {
-            console.log("[ITEM-STORE] Gracefully shutting down...");
+            logModel.date = new Date();
+            logger.log({
+                level: 'info',
+                logInfos: logModel,
+                message: "Gracefully shutting down...",
+                correlationId: '123',
```

Comment:
we don't use the "trailing comma" pattern in our code style, could you run `npm run prettier` and verify that it gets removed? If it's still there after prettier we should fix prettier